store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -286,7 +284,17 @@ protected void serialiseSizeAndPropertyValue(final String propertyName, final Sc
             } else {
                 Object value = properties.get(propertyName);
                 //serialiseNull could be different to AccumuloStoreConstants.EMPTY_BYTES
-                bytes = (null == value) ? serialiser.serialiseNull() : serialiser.serialise(value);
+                if (null != value) {
+                    bytes = serialiser.serialise(value);
+                } else if (null != this.schema.getVisibilityProperty()) {
+                    if (this.schema.getVisibilityProperty().equals(propertyName)) {
+                        bytes = serialiser.serialise("");
```

Comment:
can you change "" to be a constant please.

Reply:
Will do